src/Pelagos/Entity/DatasetSubmission.php

```diff
@@ -2861,4 +2870,26 @@ public function setRemotelyHostedFunction($remotelyHostedFunction)
     {
         $this->remotelyHostedFunction = $remotelyHostedFunction;
     }
+
+    /**
+     * Getter for fileset entity.
+     *
+     * @return Fileset
+     */
+    public function getFileSet() : Fileset
+    {
+        return $this->fileSet;
+    }
+
+    /**
+     * Setter for fileset entity.
+     *
+     * @param Fileset $fileSet The fileset entity associated with this datasetSubmission instance.
+     *
+     * @return void
+     */
+    public function setFileSet(Fileset $fileSet) : void
```

Comment:
I'd like to see this function go away when the files entity comes along. Instead there's a function maybe called addFile(s), and removeFile(s) which makes it's own fileset. We may have to use a managing utility for this?